### ISSUE TITLE
Regression : FR language doesn't work anymore (Last working version: 4.3.0)

### ISSUE THREAD

**Issue by devopsmica** [okta/okta-signin-widget#1398]
### I'm submitting a
  - [x ] bug report
  - [ ] feature request

### Background info
Using angular and imported through yarn.

Describe your issue or request here (if necessary).
For version 4.3.1, 4.3.2, 4.3.3

![image](https://user-images.githubusercontent.com/43993351/91368375-e0472d00-e7d6-11ea-97a7-ad37413e0d7d.png)

### Expected behavior
For version 4.3.0

![image](https://user-images.githubusercontent.com/43993351/91369064-b989f600-e7d8-11ea-880a-ed0db28bc738.png)

What should have happened? 
It should get the language from the node_module folder.

### What went wrong?
Here's what the browser is trying to get :
GET https://global.oktacdn.com/okta-signin-widget/4.3.3-g5ca530e/labels/json/login_fr.json 404 
GET https://global.oktacdn.com/okta-signin-widget/4.3.3-g5ca530e/labels/json/country_fr.json 404

### Steps to reproduce
No changes to the code outside changing the version.

### Your environment

  - Okta Sign-In Widget Version: 4.3.3
  - Browser: Any
  - OS: Linux
  - Language: FR

**Comment by swiftone** [comment]
@devopsmica - Thanks for the report, I'll bring this to the attention of our internationalization team.

**Comment by swiftone** [comment]
@devopsmica - it looks like we have a bug that is dropping a hash fragment into the link paths (https://global.oktacdn.com/okta-signin-widget/4.3.3-g5ca530e/labels/json/login_fr.json fails, but https://global.oktacdn.com/okta-signin-widget/4.3.3/labels/json/login_fr.json works)

We'll be pushing an update that will fix this for future widget versions, but in the meantime you should be able to manually specify the correct paths:
```
  "assets": {
    baseUrl: 'https://global.oktacdn.com/okta-signin-widget/{your widget version}'
}
```

Thanks for bringing this to our attention, and let us know if this workaround doesn't work!

**Comment by swiftone** [comment]
internal ref: OKTA-304721

**Comment by haishengwu-okta** [comment]
@devopsmica we have fixed in `4.3.4`. Please let us know if further issues. Thank you!
